packages/babel-node/src/babel-node.js

```diff
@@ -50,6 +50,11 @@ getV8Flags(function(err, v8Flags) {
         args.unshift(arg);
         break;
 
+      case "-r":
+      case "--require":
+        args.unshift(flag);
```

Comment:
`--require` has an option according to the docu: https://nodejs.org/api/cli.html#cli_r_require_module
So using `flag` here is probably wrong and should be `arg` instead, but I haven't tested it, so correct me if I'm wrong.

Reply:
Although it seems I missed that the module is not separated by equal sign but with space in this case, so `flag` and `arg` are the same. Still I think this will not work as the preceding module name will be pushed to args whereas this here gets unshifted.

Reply:
Yes, `flag` and `arg` are equal. You propose to change the `unshift` on that ? `push` ?